.github/ISSUE_TEMPLATE/patch-release.md

```diff
@@ -32,9 +33,11 @@ On the day of release:
 - [ ] Rebase the release preparation branch on the release branch
     - [ ] Ensure release date in cue matches current date.
     - [ ] Squash the release preparation commits (but not the cherry-picked commits!) to a single
-        commit. This makes it easier to cherry-pick to master after the release. 
+        commit. This makes it easier to cherry-pick to master after the release.
 - [ ] Merge release preparation branch into the release branch
     - `git co v0.<current minor version> && git merge --ff-only prepare-v0.<current minor version>.<patch>`
+- [ ] Push the release branch to update the remote
+    - (This should close the preparation branch PR)
```

Comment:
This actually needs to be done _after_ the release workflow finishes as otherwise the download links won't work. 

Maybe we could also add the `git push` command for easy copy/pasting?